src/routes/articles.route.js

```diff
@@ -416,7 +530,44 @@ articleRouter.post(
 /**
  * @swagger
  *
- * /api/v1/article/:articleId/unlike:
+ * /api/v1/article/:articleId/rate:
+ *   post:
+ *     tags:
+ *       - article
+ *     description: users can rate a single article.
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: articleId
+ *         description: the id of the article.
+ *         in: params
+ *         required: true
+ *     request:
+ *         content:
+ *         - application/json
+ *         schema:
+ *           type: array
+ *           items:
+ *         $ref: '#/definitions/article'
+ *     responses:
+ *       200:
+ *         description: article rated
+ *       404:
+ *         description: article not found
+ *       500:
+ *         description: Database error
+ */
+articleRouter.post(
+  '/:articleId/rate',
+  checkAuthorizedUser,
+  validateRating,
+  rateArticle,
+);
+
+/**
+ * @swagger
+ *
+ * /api/v1/article/:articleId/like:
```

Comment:
Kindly refer to - https://github.com/andela/zinnia-ah-backend/pull/53/files#r277588445